.github/workflows/tests.yml

```
@@ -24,6 +24,7 @@ jobs:
       uses: docker/setup-buildx-action@v3
 
     - name: Login to GHCR
+      if: ${{ secrets.GHCR_PAT != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)